apis/leds/src/lib.rs

```diff
@@ -20,29 +29,21 @@ impl<S: Syscalls> Leds<S> {
     /// that the driver is working, as it may still fail to allocate grant
     /// memory.
     pub fn count() -> Result<u32, ErrorCode> {
-        S::command(DRIVER_ID, LEDS_COUNT, 0, 0).to_result()
+        S::command(DRIVER_NUM, LEDS_COUNT, 0, 0).to_result()
     }
 
     pub fn on(led: u32) -> Result<(), ErrorCode> {
-        S::command(DRIVER_ID, LED_ON, led, 0).to_result()
+        S::command(DRIVER_NUM, LED_ON, led, 0).to_result()
     }
 
     pub fn off(led: u32) -> Result<(), ErrorCode> {
-        S::command(DRIVER_ID, LED_OFF, led, 0).to_result()
+        S::command(DRIVER_NUM, LED_OFF, led, 0).to_result()
     }
 
     pub fn toggle(led: u32) -> Result<(), ErrorCode> {
-        S::command(DRIVER_ID, LED_TOGGLE, led, 0).to_result()
+        S::command(DRIVER_NUM, LED_TOGGLE, led, 0).to_result()
     }
 }
 
-const DRIVER_ID: u32 = 2;
-
-// Command IDs
-const LEDS_COUNT: u32 = 0;
-const LED_ON: u32 = 1;
-const LED_OFF: u32 = 2;
-const LED_TOGGLE: u32 = 3;
```

Comment:
I prefer leaving these constants at the bottom of the file. They aren't part of the public interface for this crate, and I think the public interface should be above internal implementation details.

I have the same comment for the other `apis/` crates.